tests/console/commands/test_version.py

```diff
@@ -75,6 +94,11 @@ def test_short_version_update(tester: CommandTester) -> None:
     assert tester.io.fetch_output() == "2.0.0\n"
 
 
+def test_phase_version_update(tester: CommandTester) -> None:
+    tester.execute("prerelease --next-phase")
```

Comment:
It would be good if the test failed if you remove `--next-phase` in this line (so the test checks that the option is passed correctly). Maybe, you can just mock the current version so that `--next-phase` makes a difference.

Reply:
My bad on that. This new commit will change the current version and will have different a behaviour if the `--next-phase` is not supplied.